en/docs/tutorials/create-your-first-rest-api.md

```diff
@@ -52,78 +56,81 @@ The `Statistics` REST API opens on a separate page.
 
-    3. In the **Covid19 API Connection** window, enter `covid19Client` as the **Connection Name** and click **Continue to Invoke API**.
+    2. In the **Connector** panel, `covid19Client` as the **Connection Name** and click **Continue to Invoke API**.
 
-    4. In the **Operation** drop-down list, select **getStatusByCountry** and enter details as follows in the other fields:
+       3. In the **Covid19 API Connection** window, enter `covid19Client` as the **Connection Name** and click **Continue to Invoke API**.
 
-        | **Field**                  | **Value**         |
-        |----------------------------|-------------------|
-        | **Country**                | `country`         |
-        | **Response Variable Name** | `statusByCountry` |
+       4. In the **Operation** drop-down list, select **getStatusByCountry** and enter details as follows in the other fields:
 
-    5. Click **Save**.
+           | **Field**                  | **Value**         |
+           |----------------------------|-------------------|
+           | **Country**                | `country`         |
+           | **Response Variable Name** | `statusByCountry` |
 
-    6. Now let’s extract the total case count from the response and store it in a variable. Follow this procedure:
+       5. Click **Save**.
 
-        1. Click the last **+** icon in the low-code diagram.
+       6. Now let’s extract the total case count from the response and store it in a variable. Follow this procedure:
 
-        2. Under **Statements**, select **Variable** and enter details as follows:
+           1. Click the last **+** icon in the low-code diagram.
 
-            | **Field**         | **Value**                     |
-            |-------------------|-------------------------------|
-            | **Variable Type** | `int`                         |
-            | **VariableName**  | `totalCases`                  |
-            | Expression        | `<int>statusByCountry.cases`  |
+           2. Under **Statements**, select **Variable** and enter details as follows:
 
-        3. Click **Save**.
+               | **Field**         | **Value**                     |
+               |-------------------|-------------------------------|
+               | **Variable Type** | `int`                         |
+               | **VariableName**  | `totalCases`                  |
+               | Expression        | `<int>statusByCountry.cases`  |
 
-4. To retrieve the population data, connect to the World Bank API as follows:
+           3. Click **Save**.
+
+6. To retrieve the population data, connect to the World Bank API as follows:
 
     1. Click the last **+** icon in the low-code diagram.
 
-    2. Click **Connector** and then select **World Bank API**.
+    2. Click **Connector** to add a connector. Then search for the COVID-19 connector, and click on it once it appears in the search results.
```

Comment:
This should be World Bank connector. Seems like we've mistakenly mentioned COVID-19